CONTRIBUTING.md

```diff
@@ -0,0 +1,1199 @@
+# OCA Guidelines
```

Comment:
This is redundant with https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

Reply:
The question is shall we keep the original document or convert to rst and gather with the other contributing document from the website.
Both options have pros and cons IMO but I'd rather have all documentation unified into rst (easier to manage for the long term and easier to combine in Sphinx).

Still probably something that is not urgent as current document is complete and well organized.

Whatever, I would suggest to remove this document from this repository until/if it is converted to rst in order to avoid confusion. Once/if converted we would remove the `md` document.

Reply:
I'm in favor to moving CONTRIBUTING.md here.

The format is not important, as markdown is supported by Sphinx too.

Reply:
The problem is that there's a lot of places / automations that links to the other place.

Reply:
It's easy to keep the old location as a placeholder with a link to the new one.

Reply:
OK, good idea